inc/policydeployapplication.class.php

```diff
@@ -191,16 +192,11 @@ public function unapply(PluginFlyvemdmFleet $fleet, $value, $itemtype, $itemId)
          return true;
       }
 
-      $task = new PluginFlyvemdmTask();
-      $packageName = $package->getField('package_name');
-      $packageName = ($packageName) ? $packageName : $package->getField('name');
-      if (!$task->add([
-         'plugin_flyvemdm_fleets_id'   => $fleet->getID(),
-         'plugin_flyvemdm_policies_id' => $policyData->getID(),
-         'value'                       => $packageName,
-         '_silent'                     => true,
-      ])) {
-         return false;
+      $package = new PluginFlyvemdmPackage();
+      if ($package->getFromDB($itemId)) {
+         $policyFactory = new PluginFlyvemdmPolicyFactory();
+         $removeApp = $policyFactory->createFromPolicy($policyData);
+         $removeApp->apply($fleet, $package->getField('package_name'), $package, $itemId);
```

Comment:
3rd argument must be PluginFlyvemdmPackage::class or equivalent. A string is expected, not an object.

(by the way we need to fix the docblock describing this methid in PluginFlyvemdmPolicyBase::apply() )